### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by AJBaterina**
image

Design to be follow...

### CODE REVIEW

File: PRMSU-FACULTY-MANAGEMENT-SYSTEM/src/AddFaculty.java [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+        
+        semesterCBox = new JComboBox();
+        semesterCBox.setName("DEPARTMENT");
+        semesterCBox.setBounds(30, 400, 440, 40);
+        addFacultyFrame.getContentPane().add(semesterCBox);
+        
+        
+        //Combo Box Items
+        
+        departmentCBox.addItem("DEPARTMENT");
```
Comment (Getting-into-coding) [inline PRMSU-FACULTY-MANAGEMENT-SYSTEM/src/AddFaculty.java:166]: Sa Combo Box ng department dapat nakalagay lang dun ay Civil Engineering, Mechanical Engineering, Electrical Engineering, at Computer Engineering

File: PRMSU-FACULTY-MANAGEMENT-SYSTEM/src/AddFaculty.java [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+        departmentCBox.setBounds(30, 300, 440, 40);
+        addFacultyFrame.getContentPane().add(departmentCBox);
+        
+        acadYearCBox = new JComboBox();
+        acadYearCBox.setName("DEPARTMENT");
+        acadYearCBox.setBounds(30, 350, 440, 40);
+        addFacultyFrame.getContentPane().add(acadYearCBox);
+        
+        semesterCBox = new JComboBox();
+        semesterCBox.setName("DEPARTMENT");
```
Comment (Getting-into-coding) [inline PRMSU-FACULTY-MANAGEMENT-SYSTEM/src/AddFaculty.java:159]: Nakalimutan mo ata boss palitan yung name hahaha

File: PRMSU-FACULTY-MANAGEMENT-SYSTEM/src/AddFaculty.java (hunk 1 above)
Comment (Nasamid) [reply PRMSU-FACULTY-MANAGEMENT-SYSTEM/src/AddFaculty.java:166]: and Allied

**Review by Nasamid — APPROVED** [review]

**Comment by Getting-into-coding** [comment]
Pa-amend nalang para di na gagawa ng bagong pull request

**Review by Nasamid — CHANGES_REQUESTED** [review]
please change items in departmentCBox, thanks!!

**Review by Nasamid — APPROVED** [review]

**Review by Nasamid — APPROVED** [review]

**Review by Getting-into-coding — APPROVED** [review]
nice

**Pull request merged**